modules/ding_availability/js/ding_availability.js

```diff
@@ -9,79 +9,80 @@
   // Cache of fetched availability information.
   Drupal.DADB = {};
 
-  $(document).ready(function () {
-    var settings = Drupal.settings;
-    var ids = [];
-    var html_ids = [];
-
-    // Loop through the materials given in the settings and collect
-    // HTML ids and entity_ids.
-    if (settings.hasOwnProperty('ding_availability')) {
-      $.each(settings.ding_availability, function (id, entity_ids) {
-        $.each(entity_ids, function (index, entity_id) {
-          if (Drupal.DADB[entity_id] === undefined) {
-            Drupal.DADB[entity_id] = null;
-            ids.push(entity_id);
-            html_ids.push(id);
-          }
-        });
-      });
-    }
+  Drupal.behaviors.dingAvailabilityAttach = {
```

Comment:
Please remove `Attach` to the name of the behavior? `attach` is a separate function within the behavior.